### ISSUE TITLE
Vue router parent height behaviour

### ISSUE THREAD

**Issue by quantumwebco** [fullcalendar/fullcalendar-vue#57]
Hi there

I will sort out a working code example soon but it's late on a Friday and I just wanted to log this in case anyone had a solution already. Will come back and update this over the weekend!

I am using vue router to load in single file components including a calendar component which contains the FullCalendar plugin component. 

The height prop is set to "parent". The issue I am having is that it is using the height of the parent container from the previous routes component.

If I refresh the page on the calendar it sizes as expected to the parent container so there is no overflow. If I navigate from another route it takes the height of the previous component which may be taller than the screen height. 

The only solution I have found so far is to calculate the padding, margins and height of the surrounding elements and then set the max-height of the container to that so it doesn't overflow the page.

Is there a way so that it calculates after render, or is there a way to render the child FullCalendar component after the parent component? Like to call the render method in the mounted method of the parent component?

Thanks for any help and I will set up a jsfiddle as soon as I can

**Comment by quantumwebco** [comment]
One other workaround is to set the height to 0 in the prop ```:height="0"```  then in the mounted hook wait for a split second and update the height option to parent height
```
            let calendarApi = this.$refs.fullCalendar.getApi();
            setTimeout(() => {
                calendarApi.setOption('height', 'parent');
            }, 50);
```

Is this the best/correct way of doing this?

**Comment by acerix** [comment]
Please post a [runnable, stripped-down demonstration of the bug](http://fullcalendar.io/wiki/Reduced-Test-Cases/). Would really appreciate it because the time saved reproducing will be time spent fixing.

**Comment by quantumwebco** [comment]
I already said I would come back and update it with a working example later. 

Was about to buy a full license for this as I'm building a SaaS app. Seconds thoughts now.

**Comment by quantumwebco** [comment]
@acerix Why did you just close this issue when I said I was literally working on the working example codepen. 

Why have other people got open issues with no example/codepen that aren't immediately closed? 

I posted another issue with a codepen and it's been completely ignored.

I've stopped using fullcalendar in all of my projects going forwards due to this. 

This issue was actually for a SaaS app which would have required a full license but i'm not handing over cash if I'm gonna get singled out and perfectly valid issues rejected for no reason and other issues completely ignored.
